apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -8,6 +8,7 @@ body {
 	display: flex;
 	flex-direction: column;
 	min-width: 320px;
+	height: var(--doc-height);
```

Comment:
```suggestion
	min-height: 100vh
```
and then to all wrappers you should add `flex-grow: 1`
you do not need to write any programatic functionality. css only

Reply:
This has already been implemented, but the problem persists in the developer tools

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-trackmates/assets/110524091/da29cb6e-977d-41b4-8896-6428322f76f4)

That is the body element with just min-height: 100vh